src/Publishing/PublishingProfileResultModel.cs

```diff
@@ -0,0 +1,25 @@
+//----------------------------------------------------------------------------
+// <copyright file="PublishingProfileResultModel.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//----------------------------------------------------------------------------
+namespace Vasont.Inspire.Models.Publishing
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// This class represents the results of a publishing profile query request.
+    /// </summary>
+    public class PublishingProfileResultModel
+    {
+        /// <summary>
+        /// Gets or sets the total number of publishing profiles.
+        /// </summary>
+        public long TotalCount { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resulting publishing profiles from the query request.
+        /// </summary>
+        public List<PublishingProfileModel> Translations { get; set; } = new List<PublishingProfileModel>();
+    }
+}
```

Comment:
GitHub was complaining about not having a new line at the end of that file. Please see if other files have the same and if so please have a new line at the end and push, if not it's not a big deal, I just noticed.

Reply:
Some do, some don't.  I am going to leave it as is.